autoware_launch/rviz/autoware.rviz

```diff
@@ -268,21 +268,21 @@ Visualization Manager:
               Primary Color: 174; 174; 174
           Enabled: true
           Name: Vehicle
-        - Class: rviz_plugins/MrmSummaryOverlayDisplay
+        - Class: rviz_plugins/StringStampedOverlayDisplay
           Enabled: true
-          Font Size: 10
+          Font Size: 12
           Left: 10
-          Max Letter Num: 100
-          Name: MRM Summary
-          Text Color: 25; 255; 240
+          Max Letter Num: 70
+          Name: Error Diag Graph
+          Text Color: 51; 201; 220
           Top: 10
           Topic:
-            Depth: 5
+            Depth: 1
             Durability Policy: Volatile
             History Policy: Keep Last
             Reliability Policy: Reliable
-            Value: /system/emergency/hazard_status
-          Value: false
+            Value: /logging_diag_graph/debug/error_graph
```

Comment:
https://github.com/autowarefoundation/autoware.universe/pull/9421#discussion_r1877171488
```suggestion
            Value: /logging_diag_graph/debug/error_graph_text
```

Reply:
Oops. Thank you!